.travis.yml

```diff
@@ -25,11 +25,11 @@ matrix:
        env: PYDM_CHANNEL=pydm-dev
      - python: 3.6
        env: PYDM_CHANNEL=pip
-     - python: 3.7
+     - python: 3.7.3  # Pinned due to PyDM bug with Python 3.7.4
```

Comment:
Just a note... it is not a PyDM bug... this was an issue with PyQt that got fixed at 5.13.1 (to be released).
It was holding an exception and Python 3.7.4 is less forgiving of this. Phil Thompson added a bug fix into 5.13.1 but there is no 5.13 at conda-forge, yet...

Reply:
Sorry @hhslepicka didn't mean to blame you :smile:

I'll amend the comment.